StorageUtility/Linux/StorageUtility.h

```diff
@@ -28,6 +28,7 @@ namespace vtStor
 struct sStorageAdapterProperty
 {
     bool  AtaBus;
+    bool  ScsiBus;

```

Comment:
I see. I think we need a few bus to describe Property of Adapter. At now, in order to have a simple this version, there are Scsi and Ata.
if have problem we can fix in other version.